lib/msal-core/src/utils/Constants.ts

```diff
@@ -72,6 +72,19 @@ export enum ServerHashParamKeys {
     CLIENT_INFO = "client_info"
 };
 
+/**
+ * @hidden
+ * @ignore
+ * response_type from OpenIDConnect
+ * References: https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html & https://tools.ietf.org/html/rfc6749#section-4.2.1
+ *
+ */
+export const ResponseTypes = {
+    id_token: "id_token",
+    token: "token",
+    id_token_token: "id_token token"
+};
+
```

Comment:
Is this is new, can we remove the older reference in `Constants.ts`?

Reply:
Not new, it was in UserAgentApplication, moved it to Constants.